stdlib/runtime/numbers.gr

```diff
@@ -781,11 +826,10 @@ let rec numberAddSubHelp = (x, y, isSub) => {
       },
       _ => {
         throw UnknownNumberTag
-      }
+      },
     }
   }
-},
-numberAddSubRationalHelp = (x, y, isSub) => {
+}, numberAddSubRationalHelp = (x, y, isSub) => {
```

Comment:
@marcusroberts @ospencer do we want these to be on the same line or do we want a newline after the comma?

Reply:
Either is fine. This way is a bit more obvious to me, but of course I want to bring back the `and` keyword 😛

Reply:
We've been through this before  and we agreed on this way